pkg/resolver/search.go

```diff
@@ -283,6 +266,33 @@ func formatLabels(labels map[string]interface{}) string {
 	return strings.Join(labelStrings, ",")
 }
 
+// Encode array into a single string with the format.
+//  value1,value2,...
+func formatArray(itemlist []interface{}) string {
+	keys := make([]string, len(itemlist))
+	for i, k := range itemlist {
+		keys[i] = formatItem(k)
+	}
+	sort.Strings(keys)
+	return strings.Join(keys, ",")
+}
+
+// Convert interface to string format
+func formatItem(data interface{}) string {
```

Comment:
Suggestion: Would `convertToString()` be a better name?

Reply:
Done